static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -98,9 +98,13 @@ export function getTitleSubtitleMessage(event: TimelineEvent) {
       subtitle = '';
       message = event.culprit;
     } else {
+      const issuePlatformEvent = event as TimelineIssuePlatformEvent;
```

Comment:
As mentioned in another PR review, if you change the types to include literals for `event.type` typescript will discriminate automatically here